src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.scss

```diff
@@ -441,3 +442,28 @@ li {
     font-size: 0.65rem;
   }
 }
+
+@media (max-width: 992px) {
+  .input {
+    width: 150%;
+    height: 2rem;
+  }
+
+  .text-right {
+    margin-left: 30px;
+  }
+}
+
+@media (min-width: 992px) and (max-width: 1200px) {
+  .col-lg-2 {
```

Comment:
I think it's a bad idea to override bootstrap classes, better to add an appropriate one to dom elements.
For example col-md-2